website/docs/r/postgresql_virtual_network_rule.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_postgresql_virtual_network_rule"
+sidebar_current: "docs-azurerm-resource-database-sql-virtual-network-rule"
+description: |-
+  Manages a PostgreSQL Virtual Network Rule.
+---
+
+# azurerm_postgresql_virtual_network_rule
+
+Allows you to add, update, or remove an Azure PostgreSQL server to a subnet of a virtual network.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-postgresql-server-vnet-rule"
+  location = "West US"
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "example-vnet"
+  address_space       = ["10.7.29.0/29"]
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_subnet" "subnet" {
+  name                 = "example-subnet"
+  resource_group_name  = "${azurerm_resource_group.example.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet.name}"
+  address_prefix       = "10.7.29.0/29"
+  service_endpoints    = ["Microsoft.Sql"]
+}
+
+resource "azurerm_postgresql_server" "postgresql_server" {
+  name                = "postgresql-server-1"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  sku {
+    name = "B_Gen4_2"
+    capacity = 2
+    tier = "Basic"
+    family = "Gen4"
+  }
+
+  storage_profile {
+    storage_mb = 5120
+    backup_retention_days = 7
+    geo_redundant_backup = "Disabled"
+  }
+
+  administrator_login = "psqladminun"
+  administrator_login_password = "H@Sh1CoR3!"
+  version = "9.5"
+  ssl_enforcement = "Enabled"
+}
+
+resource "azurerm_postgresql_virtual_network_rule" "postgresqlsql_virtual_network_rule" {
+  name                = "postgresql-vnet-rule"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  server_name         = "${azurerm_postgresql_server.postgresql_server.name}"
+  subnet_id           = "${azurerm_subnet.subnet.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the SQL virtual network rule. Changing this forces a new resource to be created. Cannot be empty and must only contain alphanumeric characters and hyphens. Cannot start with a number, and cannot start or end with a hyphen.
```

Comment:
Can we move the `Changing this forces a new resource to be created.` to the end of the sentence?

*  `name` - (Required) The name of the SQL virtual network rule. Cannot be empty and must only contain alphanumeric characters and hyphens. Cannot start with a number, and cannot start or end with a hyphen. Changing this forces a new resource to be created.